src/framework/uicomponents/qml/MuseScore/UiComponents/FlatButton.qml

```diff
@@ -84,7 +84,8 @@ FocusScope {
         enabled: root.enabled && root.visible
 
         accessible.role: MUAccessible.Button
-        accessible.name: root.text
+        accessible.name: root.toolTipTitle
```

Comment:
it won't work for buttons that don't have a tooltip like OK/Cancel, etc.
I think we need to keep the master version
accessible.name: Boolean(root.text) ? root.text : root.toolTipTitle

Reply:
Good idea, thanks! I see you implemented it already.